src/main/java/address/prefs/PrefsManager.java

```diff
@@ -0,0 +1,94 @@
+package address.prefs;
+
+import address.events.EventManager;
+import address.events.MirrorLocationChangedEvent;
+import address.events.SaveLocationChangedEvent;
+
+import java.util.prefs.Preferences;
+import java.io.File;
+
+/**
+ * Manages saving/retrieving of preferences in the registry.
```

Comment:
I'm not too sure about the convention for this one. But i think it will be useful for developer to know if the class is a singleton class through the class descriptor.